homework/04LessonFour.js

```diff
@@ -87,7 +96,7 @@ var nestedObjects = {
 // What is an associative array? What is the difference between an array and an associative array?
 //
 // Your Answer Goes Here:
-//
+// Associative arrays are different from arrays because they can have multipe keys that are associated with values rather than in index
 //
```

Comment:
Homework 4 looks good!!! Cant think of any further feedback...Very straightforward questions and responses